jsf/jsf-widgets/src/java/org/sakaiproject/jsf/renderer/ViewRenderer.java

```diff
@@ -77,6 +84,14 @@ public void encodeBegin(FacesContext context, UIComponent component) throws IOEx
 
             String lang = locale.getLanguage();
 
+            String thisUser = SessionManager.getCurrentSessionUserId();
+            PreferencesService preferencesService = ComponentManager.get(PreferencesService.class);
+            Preferences prefs = preferencesService.getPreferences(thisUser);
+            String userTheme = "sakaiUserTheme-notSet";
```

Comment:
I don't have a problem with this pattern, we could maybe even use it to provide hints in the UI to check out dark mode. I would like @fostersdesign to verify it doesn't have negative impact on the backend work he did with the profile setting data.

Reply:
I took the entire approach straight from how the <html> tag was constructed in the portal.  If we want a different approach I would think we would change it in the portal too.